go/vt/vttablet/endtoend/transaction_test.go

```diff
@@ -203,7 +207,16 @@ func TestAutoCommit(t *testing.T) {
 	}
 }
 
+func loadTabletEnvFlags() {
```

Comment:
i think we should make this a function in a package under `tabletenv` (`tabletenvtest`?) similar to [`tabletconntest`](https://github.com/vitessio/vitess/pull/11010/files#diff-17a621e19795298d33fa2a2bded2c4707bd2e7b33817355ebd48d0a2171e3eddR795-R803) provides

Reply:
Currently it is called in two places.  I had thought of extracting it out so that it could be used from a common library but didn't see any obvious place right now.

`tabletconntest` does not export any functions: in our case we want this one function to be exported and used in vtexplain and vttablet/endtoend tests. Not sure if it is worth creating a new package just for this.

Maybe I could export this as a Test-only function inside tabletenv itself? We have precedence of such functions in other packages for use in tests only ... wdyt?

Reply:
This is not true, it exports `SetProtocol` which is what I linked

Reply:
Sorry,  that call was at the end of the file  didn't navigate all the way down. OK, added the new package as suggested.